packages/scripts/src/commands/e2e-data/snippets.ts

```diff
@@ -0,0 +1,18 @@
+// NB: Might a Map be better in this case?
```

Comment:
I think this is mostly just a case of personal preference. I know some people have a strong opinion on named vs default exports (and how to go about it) - probably the best write-up I know in favour of named is https://basarat.gitbook.io/typescript/main-1/defaultisbad. But what you have here should still fully support type inference so I think fine either way